Main.java

```diff
@@ -9,7 +10,32 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+     Scanner input = new Scanner(System.in);
+
+     final int SMALL_PIZZA = 10;
+     final int MEDIUM_PIZZA = 20;
+     final int LARGE_PIZZA = 40;
+     
+     
+     System.out.println("Enter Cm");
+     int Cm =input.nextInt();
```

Comment:
Remember, variables should always be named using camelCase and describe what they are storing or being used for. This would be better called something like "diameter" or "pizzaSize"